.github/workflows/push-helm.yml

```diff
@@ -0,0 +1,29 @@
+name: Push Helm
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
```

Comment:
Remove the block `pull_request`; I'll update the others to only run from master.